student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    @classroom = classroom
+    super(age, name, parent_permission: parent_permission)
```

Comment:
[OPTIONAL]
- [ ] Here parent_permission in the super method should then be
```suggestion
    super(age, name, parent_permission)
```